pkg/localize/locales/en/cmd/kafka.en.toml

```diff
@@ -568,7 +568,7 @@ $ rhoas kafka topic create --name topic-1
 '''
 
 [kafka.topic.create.error.topicNameIsRequired]
-one = 'topic name is required. Run "rhoas kafka topic create --name <topic-name>"'
+one = 'topic name is required. Run "rhoas kafka topic create --name topic-name"'
```

Comment:
Same as above

Reply:
<topic-name> was used as a placeholder value. If we need to remove that we should use a more realistic instance name instead of just "topic-name".

Reply:
to make it clear. Ignore my comment and @craicoverflow ask for `topic-name`=>`my-topic`